BridgeEmulator/web-ui/package.json

```diff
@@ -4,6 +4,11 @@
   "description": "",
   "keywords": [],
   "main": "src/index.js",
+  "proxy": {
```

Comment:
to help the development you can change the IP here to the one of your Hue Emulator, then change the key in `config.json` and run `npm start`.